presto-hive/src/main/java/com/facebook/presto/hive/HivePageSource.java

```diff
@@ -20,17 +20,23 @@
 import com.facebook.presto.spi.block.Block;
 import com.facebook.presto.spi.block.BlockBuilder;
 import com.facebook.presto.spi.block.BlockBuilderStatus;
+import com.facebook.presto.spi.block.ColumnarArray;
+import com.facebook.presto.spi.block.ColumnarMap;
+import com.facebook.presto.spi.block.ColumnarRow;
```

Comment:
Statically import the utility methods you used from these three classes, and remove import of the classes themselves.